duckduckgo.pot

```diff
@@ -628,6 +633,16 @@ msgstr ""
 msgid "Belgium (nl)"
 msgstr ""
 
+# smartling.placeholder_format=C
+msgctxt "settingsvalue"
+msgid "Below Snippet"
```

Comment:
A comment might be needed for this as well. Translators without context are likely to mistranslate this.